docs/api-reference/logger.md

```diff
@@ -1,6 +1,8 @@
 ## Log Dataset, Hyperparams & Metrics
 
 ```eval_rst
+.. meta::
+   :description: Use Jovian to track hyperparameters, metrics and dataset.
```

Comment:
hyper-parameters
(change in other places too)

Reply:
I think the general usage is `hyperparameters` including wikipedia. Would it be good for SEO, if we change it ?

Reply:
right. let's use hyperparameters then.